lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButtonsRow.kt

```diff
@@ -95,45 +91,81 @@ fun OdsIconToggleButtonsRow(
     }
 }
 
-data class OdsIconToggleButtonsRowItem(
-    val icon: Painter,
-    val iconDescription: String,
-    val enabled: Boolean = true
-)
+/**
+ * An icon of an [OdsIconToggleButtonsRow].
+ */
+class OdsIconToggleButtonsRowIcon : OdsComponentIcon {
 
-@Composable
-private fun IconToggleButtonsRowItem(
-    index: Int,
-    iconToggleButton: OdsIconToggleButtonsRowItem,
-    selected: Boolean,
-    displaySurface: OdsDisplaySurface,
-    onClick: (Int) -> Unit
-) {
-    val iconTint = buttonToggleIconColor(displaySurface, selected, iconToggleButton.enabled)
-    val backgroundAlpha = if (selected && iconToggleButton.enabled) 0.12f else 0f
-    val iconToggleButtonStateDescription = selectionStateDescription(selected = selected)
-
-    Icon(
-        modifier = Modifier
-            .background(
-                color = buttonToggleBackgroundColor(displaySurface)
-                    .copy(alpha = if (iconToggleButton.enabled) animateFloatAsState(backgroundAlpha, label = "").value else backgroundAlpha)
```

Comment:
We should keep the `animateFloatAsState` stuff, otherwise icon selection is not animated.

Same comment for `animateColorAsState` below.

Reply:
I removed these animations cause I had a runtime error:
```
androidx.compose.runtime.ComposeRuntimeError: Compose Runtime internal error. Unexpected or incorrect use of the Compose internal runtime API (Missed recording an endGroup). Please report to Google or use https://goo.gle/compose-feedback
```
I think this comes maybe from the `tint` attribute which should return a `Color` but if we add `animateColorAsState`, there is an animation to manage... Problem is here.